src/include/network/lan/address.rb

```diff
@@ -1080,9 +1035,9 @@ def AddressDialog
 
       if ret != :back && ret != :abort
         # general tab
-        LanItems.startmode = Ops.get_string(@settings, "STARTMODE", "")
-        LanItems.mtu = Ops.get_string(@settings, "MTU", "")
-        LanItems.firewall_zone = firewall_zone.store_permanent if firewalld.installed?
+        builder.set(option: "STARTMODE", value: Ops.get_string(@settings, "STARTMODE", ""))
+        builder.set(option: "MTU", value: Ops.get_string(@settings, "MTU", ""))
+        builder.set(option: "ZONE", value: firewall_zone.store_permanent) if firewalld.installed?
```

Comment:
hmm, I am still not much fan of this set which is just hidden `builder["STARTMODE"] = Ops.get_string(@settings, "STARTMODE", "")` so basically hash like API. I still prefer something like `builder.startmode = Ops.get_string(@settings, "STARTMODE", "")`

Reply:
no problem with that, already done something similar for the Interface and HwInfo classes and make sense even here. I just wanted isolate all cases where the builder should be used instead of (e.g.) direct access to LanItems internals. For that purpose was hashlike api enough